js/client-battle-tooltips.js

```diff
@@ -212,14 +212,29 @@ var BattleTooltips = (function () {
 
 				text += '' + (move.desc || move.shortDesc) + '</p>';
 
+				if (this.battle.gameType === 'doubles') {
+					if (move.target === 'allAdjacent') {
+						text += '<p class="movetag">&bull; Hits both foes and ally.</p>';
+					} else if (move.target === 'allAdjacentFoes') {
+						text += '<p class="movetag">&bull; Hits both foes.</p>';
+					}
+				} else if (this.battle.gameType === 'triples') {
+					if (move.target === 'allAdjacent') {
+						text += '<p class="movetag">&bull; Hits adjacent foes and allies.</p>';
+					} else if (move.target === 'allAdjacentFoes') {
+						text += '<p class="movetag">&bull; Hits adjacent foes.</p>';
+					} else if (move.target === 'any') {
+						text += '<p class="movetag">&bull; Can target distant Pok&eacute;mon in a Triples battle.</p>';
+					}
+				}
 				if ('defrost' in move.flags) {
-					text += '<p>The user thaws out if it is frozen.</p>';
+					text += '<p class="movetag">&#x2713; Thaws the user out if it is frozen.</p>';
```

Comment:
We only use checkmarks for certain move flags, not all of them. I intentionally don't use them for defrost/protect/authentic.